code/game/objects/items/cosmetics.dm

```diff
@@ -107,6 +107,8 @@
 	flags_1 = CONDUCT_1
 	w_class = WEIGHT_CLASS_TINY
 
+/var/proximity = TRUE

```

Comment:
bruh

Reply:
what do you suggest then? im open for anything